### PR TITLE
Update social badges

### PR DESCRIPTION
**Pull request by ChristopheDujarric**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -8,8 +8,11 @@
 </p>
 
 [![Bump.sh](https://img.shields.io/badge/Bump.sh-https://bump.sh-025fd7.svg)](https://bump.sh)
-[![Twitter](https://img.shields.io/badge/Twitter-@bump_hq-9cf.svg)](https://twitter.com/bump_hq)
-[![LinkedIN](https://img.shields.io/badge/LinkedIn-@bump_sh-lightgray.svg)](https://www.linkedin.com/company/bump-sh)
+[![LinkedIn](https://img.shields.io/badge/LinkedIn-@bump_sh-lightgray.svg)](https://www.linkedin.com/company/bump-sh)
+![Twitter Follow](https://img.shields.io/twitter/follow/bump_hq?style=social)
+![Mastodon Follow](https://img.shields.io/mastodon/follow/109307599957352322?domain=https%3A%2F%2Ffosstodon.org%2F&style=social)
+![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/Bump_sh?style=social)
+![YouTube Channel Subscribers](https://img.shields.io/youtube/channel/subscribers/UCefmjw--xYUL8I9feD4ZPpg?style=social)
```
Comment (antlio) [inline profile/README.md]: I'd be more in favor to keep it close to the branding and juste having the badge, wdyt?

```suggestion
[![LinkedIn](https://img.shields.io/badge/LinkedIn-@bump_sh-blue)](https://www.linkedin.com/company/bump-sh)
[![Twitter Follow](https://img.shields.io/badge/Twitter-@bump_hq-blue.svg)]
(https://twitter.com/bump_hq)
[![Mastodon Follow](https://img.shields.io/badge/Mastodon-@bump_hq-blue.svg)]
(https://fosstodon.org/@Bump_hq)
[![Reddit Follow](https://img.shields.io/badge/Reddit-@bump_hq-blue.svg)]
(https://www.reddit.com/r/Bump_sh)
[![YouTube Follow](https://img.shields.io/badge/Youtube-@bump_hq-blue.svg)]
(https://www.youtube.com/channel/UCefmjw--xYUL8I9feD4ZPpg)
```

Comment (ChristopheDujarric) [reply profile/README.md]: I was thinking that having the recognizable branding of each network, when available was cool, but it's just my opinion 😄

Comment (antlio) [reply profile/README.md:15]: It reminds me of the social plugins in 2010's blog 🥹

Comment (paulRbr) [reply profile/README.md]: I agree with Antho, I also prefer the badges without social logos :)

Comment (ChristopheDujarric) [reply profile/README.md]: OK, I surrender :D

**Pull request merged**